### PR TITLE
Adding support for cjs and mjs

### LINKED ISSUE #2: Add support for tsx cjs mjs
> Looks great! I love the minimal style. Right now the lack of .tsx file support makes it feel out of place, but once that is added I'd love to use it.
>
> Also, consider adding cjs and mjs to use the same icon as JS!

### PR DESCRIPTION
**Pull request by KyleMcKell**
Resolves #2 

Added the `js` icon to `.cjs` and `.mjs`

See image below

Picture of CJS and MJS using the JS file icon

### CODE REVIEW

File: src/symbol-icon-theme.json [hunk 1]
```diff
@@ -273,6 +275,8 @@
     "css": "brackets-sky",
     "test": "code-orange",
     "js": "js",
+    "mjs": "mjs",
+    "cjs": "cjs",
```
Comment (misolori) [inline src/symbol-icon-theme.json:279]: I don't think you need to re-define additional variants, you can just do:

```suggestion
    "mjs": "js",
    "cjs": "js",
```

**Review by misolori — CHANGES_REQUESTED** [review]
👏 Appreciate the PR! One minor nitpick, I think you can remove the mjs/cjs definitions and point to the same one since they don't change.

**Comment by KyleMcKell** [comment]
You're entirely, correct! Looking forwards to seeing this icon set grow 💜

**Comment by misolori** [comment]
✨  Beautiful, thank you!

**Pull request merged**